docs/pages/build/eas-builds-in-5-minutes.md

```diff
@@ -0,0 +1,169 @@
++            keyPassword credentialsJson["android"]["keystore"]["keyPassword"]
++        }
+     }
+     buildTypes {
+         debug {
+             signingConfig signingConfigs.debug
+         }
+         release {
+-            // Caution! In production, you need to generate your own keystore file.
+-            // see https://facebook.github.io/react-native/docs/signed-apk-android.
+-            signingConfig signingConfigs.debug
++            signingConfig signingConfigs.release
+             minifyEnabled enableProguardInReleaseBuilds
+             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+         }
+```
+
+We're defining here a new signing config (`release`) and using it for the `release` build type. The code in the `release` signing config is reading the keystore configration from the file named `credentials.json`. You don't need to create this file manually. It'll be automatically created on our build servers and populated with the credentials you generate (or provide) with Expo CLI. Using your own `credentials.json` is an advanced feature of EAS Builds. You can learn more about it in the [Advanced Credentials](https://todo) page.
+
+After making the changes to the `build.gradle` file, commit them:
+
+- `git add android/app/build.gradle`
+- `git commit -m "Configure Android project"`
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/05-configure-android.png" /></center>
+
+### Running the Build
+
+Building the Android app binary is simple - just run `expo eas:build --platform android` and you'll be guided through generating or providing your own keystore. For the sake of simplicity in this tutorial, select `Generate new keystore` and hit `ENTER`.
+
+**Warning: If you've previously set up credentials for an app with the same slug, Expo CLI will try to reuse them.**
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/06-generate-keystore.png" /></center>
+
+The build should start soon after that. Expo CLI will print a URL to the page where you can monitor the build (and view logs). Open the URL in a browser and you should see a page like this:
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/07-build-progress.png" /></center>
+<center><img src="/static/images/eas-builds/5-minute-tutorial/08-build-logs.png" /></center>
+
+Once the build completes you can download the app binary by either visiting the build status page (and clicking the download button), or by opening the URL printed by Expo CLI:
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/09-build-completed-website.png" /></center>
+<center><img src="/static/images/eas-builds/5-minute-tutorial/10-build-completed-cli.png" /></center>
+
+## Building for iOS
+
+### Set the Bundle Identifier
+
+Open `app.json` and set the bundle identifier (`expo.ios.bundleIdentifer`) that will be used to identify your application on the Apple App Store.
```

Comment:
i think we can leave this step out now that we read the bundle identifier from the xcode project right?

Reply:
Yes... and no :D Even though we read the bundle identifier from the Xcode project, the user has the ability to set his own id without opening Xcode - via app.json. As far as I remember, the default bundle identifier in the Xcode project generated with `expo init` is `org.name.PROJECT_NAME` and this isn't the greatest default that I can think of. If someone followed this tutorial and used the same name for the project (`abcd`) the credentials generation step would fail because I own the `org.name.abcd` app on the Apple App Store :)

Reply:
I'll describe this thoroughly in the `iOS` page.